src/extension/features/budget/display-total-monthly-goals/index.js

```diff
@@ -0,0 +1,109 @@
+import { Feature } from 'toolkit/extension/features/feature';
+import { getEmberView } from 'toolkit/extension/utils/ember';
+import { formatCurrency } from 'toolkit/extension/utils/currency';
+import { getCurrentRouteName } from 'toolkit/extension/utils/ynab';
+
+export class DisplayTotalMonthlyGoals extends Feature {
+  shouldInvoke() {
+    return getCurrentRouteName().indexOf('budget') !== -1 && this.settings.enabled !== '0';
```

Comment:
you should only need to perform the first part of this check. we won't even call this function if the feature isn't enabled :D